src/lib.rs

```diff
@@ -1009,56 +1008,54 @@ https://bheisler.github.io/criterion.rs/book/faq.html
             self.config.warm_up_time = dur;
         }
         if matches.is_present("measurement-time") {
-            let num_seconds =
-                value_t!(matches.value_of("measurement-time"), f64).unwrap_or_else(|e| {
-                    println!("{}", e);
-                    std::process::exit(1)
-                });
+            let num_seconds = matches.value_of_t("measurement-time").unwrap_or_else(|e| {
+                println!("{}", e);
+                std::process::exit(1)
+            });
 
             let dur = std::time::Duration::from_secs_f64(num_seconds);
             assert!(dur.as_nanos() > 0);
 
             self.config.measurement_time = dur;
         }
         if matches.is_present("nresamples") {
-            let num_resamples =
-                value_t!(matches.value_of("nresamples"), usize).unwrap_or_else(|e| {
-                    println!("{}", e);
-                    std::process::exit(1)
-                });
+            let num_resamples = matches.value_of_t("nresamples").unwrap_or_else(|e| {
+                println!("{}", e);
+                std::process::exit(1)
+            });
 
             assert!(num_resamples > 0);
 
             self.config.nresamples = num_resamples;
         }
         if matches.is_present("noise-threshold") {
-            let num_noise_threshold = value_t!(matches.value_of("noise-threshold"), f64)
-                .unwrap_or_else(|e| {
-                    println!("{}", e);
-                    std::process::exit(1)
-                });
+            let num_noise_threshold = matches.value_of_t("noise-threshold").unwrap_or_else(|e| {
+                println!("{}", e);
+                std::process::exit(1)
+            });
 
             assert!(num_noise_threshold > 0.0);
 
             self.config.noise_threshold = num_noise_threshold;
         }
         if matches.is_present("confidence-level") {
-            let num_confidence_level = value_t!(matches.value_of("confidence-level"), f64)
-                .unwrap_or_else(|e| {
-                    println!("{}", e);
-                    std::process::exit(1)
-                });
+            let num_confidence_level = matches.value_of_t("confidence-level").unwrap_or_else(|e| {
+                println!("{}", e);
+                std::process::exit(1)
+            });
```

Comment:
I think for these you can use `matches.value_of_t_or_exit()`, right?

Reply:
Oh yeah, looks like it. @Lemmih I'll leave that to you so I don't accidentally blow away whatever state you might have, unless you want me to make teh change.

Reply:
You go ahead.

Reply:
ping @benesch

Reply:
Ok, done.